app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> str | None:
+    if not command.startswith("mv "):
+        return "Wrong command"
+
+    _, source_file, new_file = command.split()
+    new_file = new_file.rstrip("/")
```

Comment:
Do not use operation-system specific delimiters in the code, your solution should be platform-agnostic

Reply:
it also works without 9th line